pages/press.md

```diff
@@ -629,9 +629,7 @@ Are you looking to get a speaker or interview about Solid? Contact us on press@s
 
 2019-07-31 [The Verge: Brain-computer interfaces are developing faster than the policy debate around them](https://www.theverge.com/interface/2019/7/31/20747916/facebook-brain-computer-interface-policy-neuralink)
 
-2019-07-31 [Joy Online: 
-Apple joins Google, Facebook, and Twitter in data-sharing project](https://myjoyonline.com/technology/2019/july-31st/apple-joins-google-facebook-and-twitter-in-data-sharing-project.php)
-
```

Comment:
This seems to break the flow and puts those two on one line, doesn't it?

Reply:
Hmm, artefact of GitHub's annoying change suggestion feature. Fixed!